src/index.html

```diff
@@ -1,13 +1,76 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html
+  lang="en"
+  class="card"
+  data-qa="card"
+>
+  <head>
+    <meta charset="UTF-8">
+
+    <meta name="viewport" content="width=device-width,
+      user-scalable=no, initial-scale=1.0,
+      maximum-scale=1.0, minimum-scale=1.0"
+    >
+
+    <title>Product cards</title>
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+
+    <link rel="stylesheet" href="./card.css">
+  </head>
+  <body class="card__content">
+    <!-- <h1>Product cards</h1> -->
+    <div class="card__container">
+      <img
+        src="./images/imac.jpeg"
+        alt="apple-a1419-iMac-27-retina-5k"
+        width="160"
+        height="134"
+        class="card__image"
+      >
+
+      <h2 class="card__tittle">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+      <p class="card__code">Product code: 195434</p>
+
+      <div class="card__description">
+        <div class="card__stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+          </div>
+        </div>
+
+        <p class="card__review">Reviews: 5</p>
+
+        <p class="card__price">Price:</p>
+
+        <p class="card__value">$2,199</p>
+      </div>
+
+      <a
+        href="#buy"
+        class="card__buy-link"
+        data-qa="hover"
+      >buy</a>
```

Comment:
```suggestion
      <a
        href="#buy"
        class="card__buy-link"
        data-qa="hover"
      >
        buy
      </a>
```

Reply:
Thanks! I fixed this mistakes. Except for links in head, because linter doesn't allow this change.